helm/conjur-config-namespace-prep/values.yaml

```diff
@@ -10,6 +10,7 @@ authnRoleBinding:
 conjurConfigMap:
   create: true
   name: conjur-connect
+  # authnStrategy: authn-k8s/authn-jwt
```

Comment:
Should we remove it at all?

Reply:
This is to show one of this values needs to be provided. There are more places with attribute as docstring in this values.yaml files

Reply:
```suggestion
  # authnStrategy: [authn-k8s|authn-jwt]
```